cl_sii/rtc/data_models.py

```diff
@@ -47,6 +47,23 @@ def validate_cesion_seq(value: int) -> None:
         raise ValueError("Value is out of the valid range.", value)
 
 
+def validate_cesion_fecha(value: datetime, tz: tz_utils.PytzTimezone) -> None:
+    """
+    Validate value of date and time when the "cesión" happened.
+
+    :raises ValueError:
+    """
+
+    tz_utils.validate_dt_tz(value, tz)
+
+    now_tz_aware = tz_utils.get_now_tz_aware()
+
+    if not (value.date() <= now_tz_aware.date()):
+        raise ValueError(
+            'Value of "fecha_cesion_dt" must be before or equal to the current day.', value
```

Comment:
In addition to the value of `fecha_cesion_dt` (`value`), I would include the value of *current day* in the exception message so that it's clear to the user which date was used for the comparison, which is especially useful when debugging time zone--related issues.